test/e2e/page-objects/pages/account-list-page.ts

```diff
@@ -234,6 +202,35 @@ class AccountListPage {
     }
   }
 
+  async addAccount(accountType: ACCOUNT_TYPE, accountName?: string) {
```

Comment:
could you add a tsdoc for this method? thanks !

Reply:
Done